packages/fxa-settings/src/components/App/RememberPassword/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback } from 'react';
+import { Localized } from '@fluent/react';
+import { useNavigate } from '@reach/router';
+import { HomePath } from '../../../constants';
+
+type RememberPasswordProps = { email?: string; forceEmail?: string };
+
+const RememberPassword = ({ email, forceEmail }: RememberPasswordProps) => {
+  const navigate = useNavigate();
+
+  const linkTarget: string = email ? '/signin' : '/force_auth';
+  const queryParam: string = email || forceEmail || '';
+
+  const handleClick = useCallback(() => {
+    navigate(HomePath + linkTarget + `?email=` + queryParam);
+  }, [navigate, linkTarget, queryParam]);
+
+  return (
+    <div className="text-sm mt-6" data-testid="remember-password-container">
+      <Localized id="pw-remember-pw">
+        <button
+          onClick={handleClick}
+          id="remember-password"
```

Comment:
Do we need this ID?

Reply:
Probably not. I think we'll be replacing all of the functions that relied on this ID once we fully move this component over to React.

Reply:
I think it's okay to leave out from a metrics perspective but I forgot I'd actually changed some of these during our TW refactor so we could use the ID for our functional tests, which is preferable over using classes. Perhaps we should leave IDs alone when they're used for our functional tests, and we have a ticket later to check on what those should be after we no longer have to maintain both versions of pages, whether it's changing them to `data-testid` or using better selectors.